### DISCUSSION TITLE
Sentry Credentials for Self-Hosted

### DISCUSSION THREAD

**Discussion by ACPK** [plausible/analytics#505] [Self-Hosted Support]
When the Plausibe analytics code tries to post to my self-hosted Plausible backend, I get a 400 error.  When I checked the logs, it seems that Plausible uses Sentry but I didn't see any Sentry ENV in the Docker file.   

 [warn] Failed to send Sentry event.Cannot send Sentry event because of invalid DSN

**Comment by ukutaht** [comment]
@ACPK Leaving Sentry unconfigured should not result in a 400. Are there any other logs?

**Comment by patschi** [comment]
You can set a sentry DSN by using following ENV variable: `SENTRY_DSN`. Works just fine, errors are being reported to my self-hosted Sentry instance.
